Utilities/packages.config

```diff
@@ -2,9 +2,53 @@
 <packages>
   <package id="GitInfo" version="2.0.10" targetFramework="net471" developmentDependency="true" />
   <package id="MathNet.Numerics" version="4.7.0" targetFramework="net471" />
+  <package id="Microsoft.CSharp" version="4.4.0" targetFramework="net48" />
```

Comment:
Again a huge load of MS stuff added to Utilities that I did not expect. Are we sure about all this?

Reply:
It was more than I expected too. It it weren't all MS stuff, I'd be much more reluctant. 

If we want to consider an alternative telemetry service, I still have an open source license ready to go with Bugsnag.